apps/ecommerce/frontend/src/components/ResourceCard.tsx

```diff
@@ -3,61 +3,74 @@ import { useState } from 'react';
 import ResourceCardRawData from './ResourceCardRawData';
 import tokens from '@contentful/f36-tokens';
 import ResourceCardMenu from './ResourceCardMenu';
-import { ResourceCardProps } from '../types';
 import MissingResourceCard from './MissingResourceCard';
 import { useDebounce } from 'usehooks-ts';
 import useExternalResource from '../hooks/useExternalResource';
+import { ExternalResource, ExternalResourceLink } from '../types';
+import { RenderDragFn } from '@contentful/field-editor-reference/dist/types';
+import getResourceProviderAndType from '../helpers/getResourceProviderAndType';
 
-const ResourceCard = (props: ResourceCardProps) => {
-  const debouncedValue = useDebounce(props.value, 300);
-  const { hydratedResourceData, isLoading, error, errorMessage, errorStatus } =
+interface Props {
+  value: ExternalResourceLink;
+  data?: ExternalResource;
```

Comment:
this isn't used anywhere - or is this intentional?

Reply:
Yeah this is later on when we are actually populating cards with a bulk endpoint